extra-js/quickjs/spec

```diff
@@ -0,0 +1,3 @@
+VER=0+git20200906
+SRCS="https://bellard.org/quickjs/quickjs-${VER: -8:4}-${VER: -4:2}-${VER: -2}.tar.xz"
```

Comment:
Why a space behind `:`?

Reply:
You should be using `2020.09.06` as version number here, since you're trying to get `2020-09-06`. You can achieve this by using `${VER//./-}`.